app/constants/credentials.js

```diff
@@ -0,0 +1,6 @@
+export const IOS_GOOGLE_CLIENT_ID = '663889381642-iqslrisaiqqrr5dhbmmol8ahtos97v2d.apps.googleusercontent.com';
```

Comment:
don't include credentials and configs on git

Reply:
🆗